docs/connectors/morningstar/screeners/investment-screener.md

```diff
@@ -0,0 +1,63 @@
+Investment Screener connector
```

Comment:
Since these are used as Docs titles on Docs sidebar, I would remove the connector bit
```suggestion
Investment Screener
```